scripts/generate_json_docs.py

```diff
@@ -291,9 +294,8 @@ def build_type(type_id, title, contents):
 
 def clean_source_path(source):
     source_path = ''
-    for path in sys.path:
-        if path in source:
-            source_path = source.replace(path, '')
+    if ABSOLUTE_LIBRARY_PATH in source:
+        source_path = source.replace(ABSOLUTE_LIBRARY_PATH, '')
 
     return source_path
```

Comment:
So if the `gcloud` grandparent is *not* a prefix, we *want* to return an empty string?